### ISSUE TITLE
Test autoupdating on Intel Mac and Windows

### ISSUE THREAD

**Issue by nang-dev** [trypear/pearai-server-issues-public#15]
Download v1.2.0
Add ZIP of the PearAI dev version
Add ZIP to Digital ocan
Edit ENV vars:
PEARAI_APP_MAC_INTEL_ZIP="https://pearai-dev.nyc3.digitaloceanspaces.com/PearAI-latest/PearAI-darwin-x64/PearAI.zip"
PEARAI_APP_WINDOWS_ZIP="https://pearai-dev.nyc3.digitaloceanspaces.com/PearAI-latest/PearAI-windows/PearAI.zip"
LATEST_RELEASE_MAC_INTEL="1.0.0"
LATEST_RELEASE_WINDOWS="1.0.0"
Change the "updateUrl" within the v1.2.0's product.json (might be package.json) to local host

run server

Edit: https://github.com/trypear/pearai-server/blob/reset/api/app/resources/updates/update.py#L111:L111

basically we just need to make sure that the os_type is right from Intel Mac and Windows
